rust/Cargo.toml

```diff
@@ -17,8 +17,14 @@ uuid = "1"
 crossbeam = { version = "0.8.2", features = ["crossbeam-channel"] }
 anyhow = "1.0.75"
 log = "0.4.20"
+env_logger = "0.11"
 
-anni-playback = { git = "https://github.com/ProjectAnni/anni" }
+anni-playback = { git = "https://github.com/ProjectAnni/anni", rev = "9207e60" }
+anni-player = { git = "https://github.com/snylonue/anni-player", rev = "466a434" }
```

Comment:
What about transfering this repository to the organization?

Reply:
done